### DISCUSSION TITLE
Localizing picklist

### DISCUSSION THREAD

**Discussion by waran70** [YetiForceCompany/YetiForceCRM#14377] [How to] [answered]
Anyone came across localizing picklist to different languages? What suggestions?

**Comment by vovpff** [comment, marked as answer]
What is the problem? You can add picklist values as `plv_name1` and etc. After that you need to add translation of this label to module in different languages.

**Comment by garryatbrighton** [comment]
Wonder if this translation can be somehow included in workflows?

**Reply by vovpff** [reply]
What is wrong?

**Reply by garryatbrighton** [reply]
Nothing is wrong! hehe I am new so want to learn!

**Reply by vovpff** [reply]
What translation you mean. Pls give example

**Reply by garryatbrighton** [reply]
Imagine you have multilingual form but picklist are in English only.  That what waran talking about I guess, and it is interesting form me too

**Reply by vovpff** [reply]
Form where? How the workflow related to this form?

**Reply by garryatbrighton** [reply]
Web form created with webservices API. Workflows are related to records obviously. Question is how to pull translated picklist to webform upon language selection

**Reply by vovpff** [reply]
Sorry I don't know

**Reply by mariuszkrzaczkowski** [reply]
@garryatbrighton  it already supports api, when logging in, enter the language and use the method /webservice/{moduleName}/Fields

https://doc.yetiforce.com/api/#/BaseModule/Api\Portal\BaseModule\Fields::get

![image](https://user-images.githubusercontent.com/10198654/112060048-f47cbe00-8b5c-11eb-8dc0-f0b7be6db348.png)

**Reply by mariuszkrzaczkowski** [reply]
![image](https://user-images.githubusercontent.com/10198654/112060301-49203900-8b5d-11eb-88a8-93a1beb2b6f7.png)
![image](https://user-images.githubusercontent.com/10198654/112060348-5d643600-8b5d-11eb-99de-942d1a76fa91.png)
